.github/workflows/wheels.yml

```diff
@@ -69,19 +67,19 @@ jobs:
           name: wheels
           path: dist
 
-  release:
-    name: Release
-    runs-on: ubuntu-latest
-    if: "startsWith(github.ref, 'refs/tags/')"
-    needs: [linux, windows, macos]
-    steps:
-      - uses: actions/download-artifact@v3
-        with:
-          name: wheels
-      - name: Publish to PyPI
-        uses: PyO3/maturin-action@v1
-        env:
-          MATURIN_PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
-        with:
-          command: upload
-          args: --skip-existing *
+  #release:
+  #  name: Release
+  #  runs-on: ubuntu-latest
+  #  if: "startsWith(github.ref, 'refs/tags/')"
```

Comment:
Since this is only running on releases, you could keep this uncommented (just to reduce the diff)